src/tools/clippy/ui-test-dependencies/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "ui-test-dependencies"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# clippy_utils = { path = "../clippy_utils" }
```

Comment:
This crate is required if Clippy internal lints should be tested in the Rust repo. Those are just UI tests as well and unrelated to dogfood.

Those internal UI tests are currently not run, because the ``internal`` feature has to be enabled during testing. I have a patch to do this, because this caused headaches during the sync multiple times in the past. 

That being said, I think being able to test Clippy with ``--stage 0`` is more important for now.

Reply:
Got it. I don't think we can run those lints without building the compiler twice, but we could add some way to opt-in to the internal tests so that they still run in CI (maybe something with `--test-args`?).

Reply:
Worth looking into. But nothing to deal with in this PR.